src/diracx/db/__main__.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+import argparse
+import asyncio
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
```

Comment:
I think it's worth adding a parameter to specify a DB already

Reply:
At the moment it's implicitly done by controlling which environment variables are set. As we know we want to use something like Alembic I wasn't sure it was worth making this more complex until we know how best to integrate it?

Reply:
I was hesitating yesterday before writing the comment exactly because of Alambic. However, it does mean that before we support for real the first endpoints (i.e. we can run a given functionality with `diracx` only, we need to have Alambic strategy sorted out). I am fine with this though, but we should have it on the road map

Reply:
I thought Alambic was already on the roadmap.

>  it does mean that before we support for real the first endpoints (i.e. we can run a given functionality with diracx only, we need to have Alambic strategy sorted out)

Why is this the case?

Reply:
> I thought Alambic was already on the roadmap.
Of course it is, the questions is just where to place it  :-) 

>Why is this the case?
Because if we start with an alternative handcooked solution, we take the usual risk of not investing the effort of doing things properly

Reply:
I don't see how adding a CLI parameter to specify the DB makes the sitution any better. It just makes it more likely it gets push off to "later".

Reply:
That is exactly my point :-) So we agree, it just means that we need to start looking at Alambic earlier than I had though

Reply:
Agreed https://github.com/DIRACGrid/diracx/issues/88